manifests/base.pp

```diff
@@ -74,15 +56,14 @@
       # all configuration is under the rsyslog.d directory
       |EOT
 
-    $_require = $rsyslog::manage_package ? {
```

Comment:
This was much better than merging into `*`, but I've updated it to keep everything consistent.